gnc/sub8_thruster_mapper/nodes/mapper.py

```diff
@@ -48,13 +49,24 @@ def __init__(self):
         self.B = self.generate_B(self.thruster_layout)
         self.min_thrusts, self.max_thrusts = self.get_ranges()
 
+        #self.kill_listener = AlarmListener('kill', self.kill_cb)
+        # Not the proper way to do this at all, but the kill listener class is being wonky right now
+        self.kill_listener = rospy.Subscriber('/alarm_raise', Alarm, self.kill_cb)
```

Comment:
You are very right, the node wasn't running. My bad, I assumed it was in the master launch and never verified. But when I added it and started running it the alarm node gave me a bunch of errors so let me fix those and I will update this again. 

I will PR the alarm stuff separately. 

Getting closer still!

Also, weird reason for it not building it that is says "mil_c3_tgen" doesn't have a CMake. Looking into it.